src/panels/lovelace/cards/hui-alarm-panel-card.ts

```diff
@@ -81,6 +82,17 @@ class HuiAlarmPanelCard extends LitElement implements LovelaceCard {
     this._code = "";
   }
 
+  protected updated(changedProps: PropertyValues): void {
+    super.updated(changedProps);
+    if (!this._config || !this.hass) {
+      return;
+    }
+    const oldHass = changedProps.get("hass") as HomeAssistant | undefined;
+    if (!oldHass || oldHass.themes !== this.hass.themes) {
```

Comment:
Should also be added to `shouldUpdate`, it will not come here because `shouldUpdate` will return false.

Reply:
And we should also act when `this._config.theme` changes

Reply:
so
```yaml
const oldConfig = changedProps.get("_config");
if (!oldHass || !oldConfig || !oldHass.themes !== this.hass.themes || oldConfig.theme !== this._config.theme)
```
?

Reply:
Yes